extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/telemetry/TelemetryConstants.java

```diff
@@ -13,4 +13,85 @@ private TelemetryConstants() {
     public static final String CONNECTION_ENDPOINT_ATTR_KEY = "connection.endpoint.id";
     public static final String CONNECTION_CLIENT_ATTR_KEY = "connection.client.id";
 
+    /**
+     * Counts all the WebSockets client opened connections.
+     */
+    public static final String CLIENT_CONNECTION_OPENED = "quarkus.websockets.client.connections.opened";
+
+    /**
+     * Counts all the WebSockets client opened connections.
+     */
+    public static final String SERVER_CONNECTION_OPENED = "quarkus.websockets.server.connections.opened";
+
+    /**
+     * Counts number of times that opening of a WebSocket server connection resulted in error.
+     */
+    public static final String SERVER_CONNECTION_OPENED_ERROR = "quarkus.websockets.server.connections.opened.errors";
+
+    /**
+     * Counts number of times that opening of a WebSocket client connection resulted in error.
+     */
+    public static final String CLIENT_CONNECTION_OPENED_ERROR = "quarkus.websockets.client.connections.opened.errors";
+
+    /**
+     * Counts all the WebSockets client closed connections.
+     */
+    public static final String CLIENT_CONNECTION_CLOSED = "quarkus.websockets.client.connections.closed";
+
+    /**
+     * Counts all the WebSockets client closed connections.
+     */
+    public static final String SERVER_CONNECTION_CLOSED = "quarkus.websockets.server.connections.closed";
+
+    /**
+     * Counts all the WebSockets server endpoint errors.
+     */
+    public static final String SERVER_MESSAGES_COUNT_ERRORS = "quarkus.websockets.server.messages.count.errors";
+
+    /**
+     * Counts all the WebSockets client endpoint errors.
+     */
+    public static final String CLIENT_MESSAGES_COUNT_ERRORS = "quarkus.websockets.client.messages.count.errors";
+
+    /**
+     * Number of messages sent and received by server endpoints.
+     */
+    public static final String SERVER_COUNT = "quarkus_websockets_server_count";
```

Comment:
Why do we sometimes use dots and sometimes underscores as delimiters in metric names?